playground/eShopLite/CatalogService/Program.cs

```diff
@@ -1,10 +1,17 @@
 using CatalogDb;
 using CatalogService;
+using Microsoft.EntityFrameworkCore;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.AddServiceDefaults();
-builder.AddNpgsqlDbContext<CatalogDbContext>("catalogdb");
+
+var connectionString = builder.Configuration.GetConnectionString("catalogdb")
+    ?? throw new InvalidOperationException("Connection string is not configured.");
+
+// Configure Npgsql.EntityFrameworkCore.PostgreSQL services
+builder.Services.AddDbContextPool<CatalogDbContext>(dbContextOptionsBuilder => dbContextOptionsBuilder.UseNpgsql(connectionString))
+    .EnrichNpgsqlEntityFrameworkCore<CatalogDbContext>(builder);
```

Comment:
In the samples it would be better to call `AddDbContextPool` after `EnrichNpgsqlEntityFrameworkCore` or whatever it ends up being called. That way conflicting configuration specified in the `AddDbContextPool` lambda will override Aspire defaults.